classes/class3/homework_readable/views/add.handlebars

```diff
@@ -0,0 +1,8 @@
+<h1> Added cat:</h1>
+
+
+ <h2>{{cat.cat_name}}</h2>
+ <ul>
+ <li>Age: {{cat.age}}</li>
```

Comment:
When we get into larger html files, you'll want to make sure you always indent nested tags to keep things readable.